en/components/toast.md

```diff
@@ -124,42 +124,10 @@ public show(toast) {
 
 <div class="divider--half"></div>
 
-### API Summary
-The following tables summarize some of the useful **igx-toast** component inputs, outputs and methods.
-
-<div class="divider--half"></div>
-
-#### Inputs
-The following inputs are available in the **igx-toast** component:
-
-| Name | Type | Description |
-| :--- | :--- | :--- |
-| `autoHide` | boolean | Specifies whether the toast is automatically hidden after `displayTime`. By default, `autoHide` is set to **true**. |
-| `displayTime` | number | Specifies the interval in milliseconds in which the toast is visible. By default, `displayTime` is set to 4000 milliseconds. |
-| `isVisible` | boolean | Specifies the component visual state. By default, `isVisible` is set to **false**. |
-| `message` | string | Specifies the displayed message. |
-| `position` | `IgxToastPosition` | Specifies the toast position. Possible options are `IgxToastPosition.Top`, `IgxToastPosition.Middle`, `IgxToastPosition.Bottom`. By default, `position` is set to `IgxToastPosition.Bottom`. |
-
-<div class="divider--half"></div>
-
-#### Outputs
-The following outputs are available in the **igx-toast** component:
-
-| Name | Description |
-| :--- | :--- |
-| `onShowing`  | Emits an event prior to the toast being shown. |
-| `onShown`  | Emits an event when the toast is shown. |
-| `onHiding`  | Emits an event prior to the toast being hidden. |
-| `onHidden`  | Emits an event when the toast is hidden. |
-
-#### Methods
-The following methods are available for the **igx-toast** component:
-
-| Name | Description |
-| :--- | :--- |
-| `show()`  | Shows the toast component and hides it after `displayTime` if `autoHide` is set to **true**. |
-| `hide()`  | Hides the toast component. |
+### API References
+In this article we learned how to use and configure the [`IgxToastComponent`]({environment:angularApiUrl}/classes/igxtoastcomponent.html). For more details in regards its API, take a look at the links below:
 
+* [`IgxToastComponent`]({environment:angularApiUrl}/classes/igxtoastcomponent.html)
```

Comment:
Provide Sassdoc reference to the [igx-toast-theme](https://staging.infragistics.local/products/ignite-ui-angular/docs/sass/#function-igx-toast-theme)